CHANGELOG.md

```diff
@@ -7,20 +7,24 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
```

Comment:
I think your editor changed more than we want it to :)

Reply:
Oh I pull it from main to update the branch and it just changed the `Main (unreleased)` into a title, do we want to remove that and keep the `Main (unreleased)` as a normal text?

Reply:
```
Main (unreleased)
-----------------
``` 

is a title (it's special Markdown syntax), we did it that way because it's easier to differentiate with the other headers when you're reading as plaintext :) 

I'd ideally like to keep the formatting for the releases as-is to maintain the readability

Reply:
Makes sense, and also it is my editor being annoying by auto formatting everything on save. Just pushed the updated with the correct format.

Reply:
Got it. Thanks a lot for helping with the format!